src/new-client/src/App/LiveRates/Tiles/ExecutionStatusAlertSwitch/Response.tsx

```diff
@@ -123,4 +123,34 @@ const Rejected = () => (
   </ResponseContainer>
 )
 
-export { Done, Rejected }
+const ClientTimeout = () => {
+  const symbol = useContext(SymbolContext)
+  const baseTerm = useBaseTerm(symbol)
+  return (
+    <ExecutionStatusAlertContainer status={ExecutionStatus.ClientTimeout}>
+      <CurrencyPairDiv>
+        <AlignedTriangle />
+        {baseTerm}
+      </CurrencyPairDiv>
+      <TradeMessageDiv>
+        Trade execution taking longer than expected
+      </TradeMessageDiv>
+    </ExecutionStatusAlertContainer>
+  )
+}
+
+const RequestTimeout = () => {
+  const symbol = useContext(SymbolContext)
+  const baseTerm = useBaseTerm(symbol)
+  return (
+    <ExecutionStatusAlertContainer status={ExecutionStatus.RequestTimeout}>
+      <CurrencyPairDiv>
+        <AlignedTriangle />
+        {baseTerm}
+      </CurrencyPairDiv>
+      <TradeMessageDiv>Trade execution timeout exceeded</TradeMessageDiv>
+    </ExecutionStatusAlertContainer>
+  )
+}
+
+export { Done, Rejected, ClientTimeout, RequestTimeout }
```

Comment:
I'm not a big fan of having a component per status. However, since this is something that wasn't introduced by this PR, let's address it in a separate one.

Reply:
No problem, I will modify the things above and save this one for next PR, thanks for your time Josep!